cdap-docs/reference-manual/source/http-restful-api/metrics.rst

```diff
@@ -275,6 +275,76 @@ These metrics are available in a stream context:
    * - ``system.collect.bytes``
      - Number of bytes collected by the stream
 
+These metrics on request and response are available for system and user services:
```

Comment:
How about the following?
`These metrics are available for a service context, both for a system service (like transaction service), or a user service context:`